Answers.md

```diff
@@ -0,0 +1,16 @@
+## Self-Study/Essay Questions
+
+Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
+
+1. Explain the difference between `RDBMS` and `SQL`.
+    SQL is a standard language set of commands used to manage databases.RDBMS is the database itself.
+
+2. Why do tables need a `primary key`?
+    To keep track of the database entry it also gives its child databases a way to point back at it.
```

Comment:
I'm not sure 'child databases' in the best phrase to get the meaning across here, especially since the relationship could be either a _parent_ or _child_ one!